todo-angular/src/app/app.module.ts

```diff
@@ -3,16 +3,12 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { HttpClientModule } from '@angular/common/http';
```

Comment:
Remove this line

Reply:
Hvis jeg fjerner HttpClientModule, så kan den ikke rendere todo :/